sdk/communication/azure-communication-administration/src/samples/java/com/azure/communication/administration/ReadmeSamples.java

```diff
@@ -331,4 +336,75 @@ public AreaCodes getAreaCodes() {
         }
         return null;
     }
+
+    /**
+     * Sample code to create a phone number search
+     *
+     * @return PhoneNumberSearch for the phone plan
+     */
+    public PhoneNumberSearch createPhoneNumberSearch() {
+        String phonePlanId = "PHONE_PLAN_ID";
+
+        List<String> phonePlanIds = new ArrayList<>();
+        phonePlanIds.add(phonePlanId);
+
+        CreateSearchOptions createSearchOptions = new CreateSearchOptions();
+        createSearchOptions
+            .setAreaCode("AREA_CODE_FOR_SEARCH")
+            .setDescription("DESCRIPTION_FOR_SEARCH")
+            .setDisplayName("NAME_FOR_SEARCH")
+            .setPhonePlanIds(phonePlanIds)
+            .setQuantity(2);
+
+        try {
+            PhoneNumberClient phoneNumberClient = createPhoneNumberClient();
+            CreateSearchResponse createSearchResponse = phoneNumberClient.createSearch(createSearchOptions);
+
+            System.out.println("SearchId: " + createSearchResponse.getSearchId());
+            PhoneNumberSearch phoneNumberSearch = phoneNumberClient.getSearchById(createSearchResponse.getSearchId());
+
+            for (String phoneNumber
+                 : phoneNumberSearch.getPhoneNumbers()) {
+                System.out.println("Phone Number: " + phoneNumber);
+            }
+
+            return phoneNumberSearch;
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    /**
+     * Sample code to purchase a phone number search
+     */
+    public void purchasePhoneNumberSearch() {
+        String phoneNumberSearchId = "SEARCH_ID_TO_PURCHASE";
+
+        try {
+            PhoneNumberClient phoneNumberClient = createPhoneNumberClient();
+            phoneNumberClient.purchaseSearch(phoneNumberSearchId);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Sample code to configure a phone number
+     */
+    public void configurePhoneNumber() {
+        PhoneNumber phoneNumber = new PhoneNumber("PHONENUMBER_TO_CONFIGURE");
+        PstnConfiguration pstnConfiguration = new PstnConfiguration();
+        pstnConfiguration.setApplicationId("APPLICATION_ID");
+        pstnConfiguration.setAzurePstnTargetId("AZURE_PSTN_TARGET_ID");
```

Comment:
You can remove this line. This target id was accidentally added to the request object exposed to the user.

Reply:
Sure